models/fixtures/action.yml

```diff
@@ -17,8 +17,8 @@
   act_user_name: user3
   repo_id: 3
   repo_user_name: user3
-  repo_name: repo3 # TODO old or new name?
-  is_private: false
+  repo_name: repo3
+  is_private: true
```

Comment:
Why is the is_private value changed here ? The commit log doesn't say anything about this